docs/api-access.md

```diff
@@ -0,0 +1,73 @@
+# Handling access scope operations
+
+#### Table of contents
+
+- [Common ApiAccess operations](#common-apiaccess-operations)
+- [Using ApiAccess to handle changes in app access scopes](#using-apiaccess-to-handle-changes-in-app-access-scopes)
+
+There are common operations that are used for managing access scopes in apps. Such operations include serializing, deserializing and normalizing scopes. Other operations can include checking whether two sets of scopes grant the same API access or whether one set covers the access granted by another set.
+
+To encapsulate the access granted by access scopes, you can use the `ApiAccess` value object.
+
+## Common ApiAccess operations
+
+### Constructing an ApiAccess
+
+```python
+api_access = ApiAccess(["read_products", "write_orders"]) # List of access scopes
+another_api_access = ApiAccess("read_products, write_products") # String of comma-delimited access scopes
+```
+
```

Comment:
Could we show a constructor that takes in a string here as well (And show an example of an `unauthenticated` scope)